test/hotspot/gtest/opto/test_moveBits.cpp

```diff
@@ -37,10 +38,8 @@ inline void test_moveBits() {
   printf("testing %sint%d_t...\n", IS_U ? "u" : "", NBIT);
   ASSERT_EQ(reverse_bits((T)0), (T)0);
   ASSERT_EQ(reverse_bits((T)-1), (T)-1);
-  ASSERT_EQ(reverse_bytes((T)0), (T)0);
-  ASSERT_EQ(reverse_bytes((T)-1), (T)-1);
-  ASSERT_EQ(reverse_bits_in_bytes((T)0), (T)0);
-  ASSERT_EQ(reverse_bits_in_bytes((T)-1), (T)-1);
+  ASSERT_EQ(byteswap<T>((T)0), (T)0);
+  ASSERT_EQ(byteswap<T>((T)-1), (T)-1);
```

Comment:
This isn't testing moveBits any more so doesn't belong here.

Reply:
Moved test_moveBits.cpp to be in utilities and split out byteswap tests into their own file.